From 81c4f50914bd0f696a0a6c356982e97594bd2c77 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michael=20La=C3=9F?= <lass@mail.uni-paderborn.de>
Date: Tue, 19 May 2020 16:17:01 +0200
Subject: [PATCH 1/4] Temporary fix for compilation with GCC 10

See:
* https://bugs.gentoo.org/706738
* https://gerrit.openafs.org/14106

Backporting that fix to OpenAFS 1.8 requires some changes in the xstat
code so for now let's just allow duplicate declarations.

Change-Id: I0bbf317dcf584d5531db714c5e9b986f3fbb6a0b
---
 src/cf/osconf.m4 | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/cf/osconf.m4 b/src/cf/osconf.m4
index 5f0d73fc2..7cf761757 100644
--- a/src/cf/osconf.m4
+++ b/src/cf/osconf.m4
@@ -681,6 +681,8 @@ if test "x$GCC" = "xyes"; then
     else
       CFLAGS_NOSTRICT=
     fi
+  else
+    XCFLAGS="${XCFLAGS} -fcommon"
   fi
 else
   case $AFS_SYSNAME in
-- 
2.30.0

